CHANGES.md

```diff
@@ -4,6 +4,12 @@
 
 (nothing yet)
 
+## 2.4.1
```

Comment:
nit: I just realized that a 2.4.1 version hasn't been released, so I'm not sure if this should be added when the release is cut and the actual version number is bumped in package.json. Not a big deal anyway I think.

Reply:
I intend to do the release when this is in, so it saves a bit to just bump the ver now.